compat/src/render.js

```diff
@@ -161,6 +161,21 @@ options.vnode = vnode => {
 				value = undefined;
 			}
 
+			if (/^oninput$/i.test(i) && normalizedProps.oninput) {
+				const first = normalizedProps.oninput;
+				const next = value;
+				value = function onInput() {
+					const eventHandlerArguments = arguments;
+					[first, next].forEach(callback => {
+						try {
+							callback(eventHandlerArguments);
```

Comment:
Could save a few bytes

```suggestion
				value = function onInput(e) {
					[first, next].forEach(callback => {
						try {
							callback(e);
```

Reply:
Done, thanks! looks better